index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MyPortfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap">
+    <script src="https://kit.fontawesome.com/366c518b05.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="#" class="firstTitle">Welcome</a>
+            <a href="#" class="hamburgerButton"><i class="fa-solid fa-bars"></i></a>
+        </nav>
+    </header>
+    <section class="heroIntro">
+        <div class="heroIntroContainer">
+            <h1 class="heroIntroHeading">
+                Hey there. I'm<br>Troos<span class="spanText"><br>I'm a software<br>developer</span>
+                <br>
```

Comment:
[OPTIONAL] Well done overall 💯 but I'd advise not using `<br>` tag. The main reason for not using `<br>` is that it's not [semantic](http://en.wikipedia.org/wiki/Semantic_Web). If you want two items in different visual blocks, you probably want them in different logical blocks. 📦 

In most cases, this means just using different elements, for example `<p>Stuff</p><p>Other stuff</p>`, and then using CSS to space the blocks out properly. 🤔 

There are cases where `<br>` is semantically valid, i.e. cases where the line break is part of the data you're sending. This is really only limited to 2 use cases - poetry and mailing addresses. 😄